charts/metallb/templates/podmonitor.yaml

```diff
@@ -84,6 +85,7 @@ rules:
       - ""
     resources:
       - pods
+      - endpoints
```

Comment:
I don't think we need to read endpoints if we are using podmonitors

Reply:
Thanks!